src/components/menu/menu-test.stories.tsx

```diff
@@ -1,13 +1,26 @@
 import React, { useState } from "react";
 import { action } from "@storybook/addon-actions";
-
-import { Menu } from ".";
+import {
+  Menu,
+  MenuDivider,
+  MenuSegmentTitle,
+  MenuTitleProps,
+  ScrollableBlock,
+} from ".";
 import { MenuType } from "./menu.context";
-import MenuItem from "./menu-item";
+import MenuItem, { MenuWithChildren } from "./menu-item";
 import MenuFullscreen, { MenuFullscreenProps } from "./menu-full-screen";
 import Search from "../search";
 import NavigationBar, { NavigationBarProps } from "../navigation-bar";
 import GlobalHeader from "../global-header";
+import Box from "../box/box.component";
```

Comment:
Have a look at `index.ts` within `Menu`: You may be able to use these from one import instead of importing types etc from numerous different files

Reply:
`import {
  Menu,
  MenuDivider,
  MenuSegmentTitle,
  MenuTitleProps,
  ScrollableBlock,
  MenuItem,
  MenuWithChildren,
  MenuFullscreen,
  MenuFullscreenProps,
  MenuProps,
  MenuDividerProps,
  ScrollableBlockProps,
} from ".";`

Something like this